### ISSUE TITLE
Saw you changed ResNet to ConvNext

### ISSUE THREAD

**Issue by Mut1nyJD** [lucidrains/denoising-diffusion-pytorch#12]
I recently was thinking of doing the same but was wondering if you have tried it and seen any actual benefit? 
I am currently using the UNet architecture from OpenAI improved/guided diffusion as well

Thanks

**Comment by lucidrains** [comment]
@Mut1nyJD it seems to work! im just tired of batchnorm or groupnorms - layernorms all the way from now on :D

**Comment by Mut1nyJD** [comment]
@lucidrains Thanks for your honest answer. I see friends do not let friends use Batchnorm ;) 

Just wondering if you saw any improvements in image quality, convergence or sampling time
